api/app/controllers/spree/api/v2/storefront/cms_pages_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Spree
+  module Api
+    module V2
+      module Storefront
+        class CmsPagesController < ::Spree::Api::V2::ResourceController
+          private
+
+          def model_class
+            Spree::CmsPage
+          end
+
+          def resource
+            @resource ||= scope.find_by(slug: params[:slug])
```

Comment:
```suggestion
            @resource ||= scope.friendly.find(params[:slug])
```

this way we can support both slugs and IDs